bin/MixinGenerator.php

```diff
@@ -0,0 +1,237 @@
+<?php
```

Comment:
let's add this to export ignores in `.gitattributes`

Reply:
@Ocramius I need help with this - I never released composer packages myself :shrug:

Reply:
look at `.gitattributes` in ProxyManager when you got time: it is just a file 👍

Reply:
ah, there's already such a file in this repo: just adf the `bin` directory in there 👍